pkg/destroy/gcp/firewall.go

```diff
@@ -15,6 +16,19 @@ const (
 	firewallResourceName = "firewall"
 )
 
+func (o *ClusterUninstaller) deleteFirewallByName(ctx context.Context, resourceName string) error {
+	items, err := o.listFirewallsWithFilter(ctx, "items(name),nextPageToken", func(item string) bool { return strings.Contains(item, resourceName) })
```

Comment:
Same question as above: should we make it consistent as currently we use both `==` and `.Contains` in different places?